src/stores/user/subscriptions.js

```diff
@@ -93,19 +93,22 @@ export function useUserSubscriptionStatus () {
       const { loading, subscription } = data
 
       let isPro = false
+      let isProPlus = false
       let isTrial = false
       let trialDaysLeft = null
 
       if (subscription) {
         const { trialEnd, plan } = subscription
-        isPro = plan.name === PRO || plan.name === PRO_PLUS
+        isProPlus = plan.name === PRO_PLUS
+        isPro = plan.name === PRO || isProPlus
```

Comment:
If the `isProPlus` is true, `plan.name === PRO` is always false. This condition should be written as `isPro = isProPlus || plan.name === PRO`